app/assets/stylesheets/posts/_post.scss

```diff
@@ -117,6 +118,10 @@
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+
+  &__author > span {
+    color: #8f8f8f;
```

Comment:
same here, we can set this color in our `colors.scss` file

Reply:
done!